README.md

```diff
@@ -107,10 +103,19 @@ our [code of conduct](CODE_OF_CONDUCT.md) in all interactions both on and offlin
 
 If you want to report a problem or suggest an enhancement we'd love for you to
 [open an issue](../../issues) at this github repository because then we can get right on
-it. But you can also reach us on the [NeuroTechX slack #moabb
-channel][link_neurotechx_signup] where we are happy to help!
+it.
+
+For a less formal discussion or exchanging ideas, you can also reach us on the [Gitter
+channel][link_gitter] or join our weekly office hours! This an open video meeting
+happening every Thursday at [18:30 GMT+1](https://time.is/en/Paris), please ask the link
+on the gitter channel. We are also on [NeuroTechX slack #moabb
+channel][link_neurotechx_signup].
 
-## Find out more
+### Find out more
```

Comment:
What's the point of this section?
`Documentation` section already exist and links to tutorials and examples already in `Use moabb` section
Turns out that this duplicates other parts of contents...

Reply:
Yes, we could cut this section.